codes/models/alt_growth_rates/KOR_adm1.do

```diff
@@ -202,6 +202,7 @@ p_1 * _b[p_1] + ///
 p_2 * _b[p_2] + ///
 p_3 * _b[p_3] + ///  
 p_4 * _b[p_4] + ///  
+p_5 * _b[p_5] + ///  

```

Comment:
same as IRN, we need to add testing regime in y_actual and y_counter, but not in treatment

Reply:
fixed in all countries